lib/site_prism.rb

```diff
@@ -13,7 +13,7 @@ module SitePrism
   autoload :Waiter, 'site_prism/waiter'
 
   class << self
-    attr_accessor :enable_logging
+    attr_writer :enable_logging
```

Comment:
You can actually make this a ```attr_reader```

Reply:
This has a separate reader below and needs to be a writer